frontend/package.json

```
@@ -53,8 +56,9 @@
     "devDependencies": {
         "@testing-library/jest-dom": "^5.12.0",
         "@testing-library/react": "^11.2.7",
+        "axios-mock-adapter": "^1.19.0",
         "gh-pages": "^2.2.0",
-        "jest": "26.6.0",
+        "jest": "^26.6.0",
```

```suggestion
        "jest": "26.6.0",
```

I believe this is breaking the frontend pipeline, since the testing requires an exact version. Why did you change this @SaiVarunVaranasi and can we change it back?